### DISCUSSION TITLE
Woensdag 20-11-2024

### DISCUSSION THREAD

**Discussion by Annevd** [rutgerkock/buurtcampus-oost#10] [Stand-ups]
- Hoe gaat het met je en waar ben je nú mee bezig?
- Wat wil je deze week bereiken (aan werken)?
- Waar loop je tegen aan?

**Comment by Annevd** [comment]
### Hoe gaat het met je en waar ben je nu mee bezig?

Op dit moment gaat het redelijk goed. Ik ben nu bezig met de must haves uit het projectplan te verwerken van de contact pagina. 

### Wat wil je deze week bereiken (aan werken)?

Ik hoop alle must haves te verwerken deze week en eventueel alvast aandacht te besteden aan de should haves.

### Waar loop je tegen aan?

Ik loop nu aan tegen het feit dat ik de taken van het projectplan moet combineren met het werken aan de leertaken en het halen van bewijslast. Ik moet op een of andere manier nog complexe code ergens voor schrijven en nog iets van nieuwere principes op het gebied van frontend toepassen. Ook weet ik niet hoe ik tegelijkertijd ook nog het accessibility menu af moet krijgen in de tijd die we hebben. Dit kost namelijk ook nog best veel tijd, en kan ik zeker niet helemaal alleen.

**Comment by Lmikkers** [comment]
### Hoe gaat het met je en waar ben je nú mee bezig?
Goed alleen heb ik nog niet heel goed grip op wat er nog moet gebeuren, ik heb het project plan ingevuld voor de stekjes, zaden, detailpagina's, footer en Harry. Hiervoor moet ik alleen nog issues voor inschieten. 
Ik ben vooral bezig geweest met de componenten deeltaak.

### Wat wil je deze week bereiken (aan werken)?
Ik wil de issues erin gaan zetten en een begin maken van een van de pagina's

### Waar loop je tegen aan?
Ik vind het lastig om de opdracht van de sprint en de opdrachtgever opdracht samen te combineren, soms vergeet ik dat. Hier moet ik meer aandacht aan besteden, ook omdat de website in februari af moet zijn.

**Comment by Nazneen05x** [comment]
### Hoe gaat het met je en waar ben je nú mee bezig?
Vorige week heb ik samen met Anne en Wingsan, het projectplan besproken en geprioriteerd welke taken eerst af moeten voor het lanceringsfeest. Het menu staat bovenaan mijn takenlijst.

###  Wat wil je deze week bereiken (aan werken)?
Ik ben begonnen met het opnieuw bouwen van het menu. Het menu krijgt nu een dropdown menu op de desktopversie, omdat er twee nieuwe pagina’s komen en de opdrachtgever wilde graag dat het in een dropdown menu komt. Ik heb de eerste opzet van de desktop gebouwd, nu alleen het dropdown menu bouwen en dan is de desktop klaar. Maar eerst wil ik beginnen met het bouwen van het mobiele menu en daarna verder met de desktop. Deze week wil ik dat het uitklapmenu werkt. 

### Waar loop je tegen aan?
Op dit moment nog niks, maar ik heb altijd moeite met het maken van een mobiel menu dus dat gaat een klus worden!. Ik ben van plan om het met een dialog (modal) te maken. Helaas ondersteunde de popover api nog niet helemaal Safari, anders had ik die gebruikt voor het mobiel menu.

**Comment by wingsvn** [comment]
**Hoe gaat het met je en waar ben je nu mee bezig?**
Het gaat prima. De afgelopen week ben ik vooral bezig geweest met de componenten deeltaak. We hebben vorige week het projectplan besproken en taken verdeeld.

**Wat wil je deze week bereiken of waar wil je deze week aan werken?**
Ik ga nu bezig met de must haves van de agenda pagina. Dus het checken van de contrast en het toevoegen van de buttons op de agenda pagina.

**Waar loop je tegen aan?**
Het coderen zelf gaat nog een beetje stroef, omdat ik nog steeds lastig vindt om mijn weg te vinden binnen Sveltekit.
